autoload/zettel/vimwiki.vim

```diff
@@ -572,10 +578,14 @@ function! zettel#vimwiki#create(...)
   " let vimwiki to open the wiki file. this is necessary
   " to support the vimwiki navigation commands.
   " test if the current file is in wiki
-  if s:vimwiki_dir.path =~ vimwiki#path#current_wiki_file()
+  echom '1: ' . vimwiki#path#current_wiki_file()
+  echom '2: ' . s:vimwiki_dir.path[0:len(s:vimwiki_dir.path)-2]
+  if vimwiki#path#current_wiki_file() =~ s:vimwiki_dir.path[0:len(s:vimwiki_dir.path)-2]
```

Comment:
I'm now pretty sure that a) my previous PR had an issue, and kind of accidentally worked ( `"hellobello" =~ "hello"` is true, while `"hello" =~ "hellobello"` is not and a trailing slash needs to be removed). 

Now this actually does what's in the comments (i.e. badpath actually happens when we are outside of the wiki).

Now, I originally did my little fix to conform to the documentation, but I was mislead by two things: I have a plugin that placed me at the root of my wiki no matter what file I opened within the wiki and that root of the wiki happened to be the "correct" folder most of the time. But looking at this now, the code only has two paths (in wiki or not in wiki), but the documentation is a bit more complicated:

> The newly created wiki-file will be saved in:
  (1) the current directory if it is in the `g:vimwiki_list` paths
  (2) the directory-path of the first `g:vimwiki_list` with non-empty `g:zettel_options`
  (3) the directory-path given by the first `g:vimwiki_list` entry
  (4) vimwiki's default directory

Do we want to actually support (1)? I.e. support not placing it in the one single folder that is the zettelkasten?